ha/core/system_health/system_health.py

```diff
@@ -86,6 +90,145 @@ def get_status(self, component: str, component_id: str=None, **kwargs):
             Log.error(f"Failed reading status for component: {component} with Error: {e}")
             raise HaSystemHealthException("Failed reading status")
 
+    def get_status(self, component: CLUSTER_ELEMENTS = CLUSTER_ELEMENTS.CLUSTER.value, depth: int = 1, version: str = SYSTEM_HEALTH_OUTPUT_V1, **kwargs):
```

Comment:
Some debug logs will be required. Please review to see if more logs can be placed.

Reply:
Added some debug logs.